index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet"href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>To-Do-List</title>
+</head>
+<body>
+    
+<div id="myDIV" class="header">
+  <h2 class="my-to-do-list">My To Do List</h2>
+  
+  <input type="text" class="to-do-input" placeholder="Enter your task...">
+  <span  class="add-button"> <i class="fa fa-plus"></i>&ensp; Add Item</span>
```

Comment:
Clickable elements should always be a `<button />` or `<a />`

Reply:
Added button and placed the line 18 and 19 inside form.